homeassistant/components/media_player/philips_js.py

```diff
@@ -0,0 +1,206 @@
+"""
+Media Player component to integrate TVs exposing the Joint Space API.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.philips_js/
+"""
+import logging
+from datetime import timedelta
+import json
+
+import requests
+import voluptuous as vol
+
+from homeassistant.components.media_player import (
+    PLATFORM_SCHEMA, SUPPORT_SELECT_SOURCE, SUPPORT_TURN_OFF,
+    SUPPORT_VOLUME_STEP, SUPPORT_VOLUME_MUTE, MediaPlayerDevice)
+from homeassistant.const import (
+    STATE_ON, STATE_OFF, STATE_UNKNOWN, CONF_HOST, CONF_NAME)
+from homeassistant.util import Throttle
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=30)
+
+SUPPORT_PHILIPS_JS = SUPPORT_TURN_OFF | SUPPORT_VOLUME_STEP | \
+                     SUPPORT_VOLUME_MUTE | SUPPORT_SELECT_SOURCE
+
+DEFAULT_DEVICE = 'default'
+DEFAULT_HOST = '127.0.0.1'
+DEFAULT_NAME = 'Philips TV'
+DEVICE_BASE_URL = 'http://{0}:1925/1/{1}'
```

Comment:
This lead to the assumption that the port can not be changed by the user on the TV, right?